scipy/optimize/_differentialevolution.py

```diff
@@ -45,11 +45,16 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
         to ``len(x)``.
     bounds : sequence or `Bounds`
         Bounds for variables. There are two ways to specify the bounds:
-        1. Instance of `Bounds` class.
-        2. ``(min, max)`` pairs for each element in ``x``, defining the finite
-        lower and upper bounds for the optimizing argument of `func`.
+
+            1. Instance of `Bounds` class.
+            2. ``(min, max)`` pairs for each element in ``x``, defining the
+               finite lower and upper bounds for the optimizing argument of
+               `func`.
```

Comment:
To render as a list, this just needed a preceding blank line.

Fortunately, our Sphinx theme no longer renders block quotes the way it used to (gh-13106), but I think it would be better to remove the block quote indentation gradually rather than add it.

Reply:
I based the formatting off the `minimize` docstring, specifically where bounds are mentioned. I also looked at the rendered output and it seemed ok.

Reply:
Yup, but when things are indented, remember how it used to render with that weird vertical bar? 

> That's because the indent indicates to Sphinx that it's a block quote, like this.

The PR (gh-13106) was about fixing this. I closed it because it wasn't merged and the vertical bar disappeared after a style update, but it could come back. So I'm just suggesting to avoid the indentation because it's not necessary and could change to look wrong again in the future.